### ISSUE TITLE
针对不同硬件及实施不同阶段，如何调整脚本参数进行测试

### ISSUE THREAD

**Issue by pjvs1478** [verl-project/verl#6754]
## 问题背景

使用 verl + Ascend 910b 跑 GRPO 训练（Qwen2.5-0.5B-Instruct + GSM8K），原始脚本一个 epoch 需要 467 步、预计 9.5 小时。实际工作中，在不同阶段对训练的需求完全不同，但缺乏一份参数调优指引。

### 1. 冒烟测试
- **目标**：2-3 分钟内跑通 推理→reward→反向传播→权重更新 全流程，验证链路无报错
- **不关心**：收敛效果、精度
- **当前做法**：缩小 batch_size、缩短序列长度、关掉验证、限制 10 步

### 2. Overfit 测试
- **目标**：在小数据集上过拟合，确认模型有能力学到东西、loss 能下降
- **期望**：val acc 到接近 100%

### 3. 显存调优
- **目标**：在固定硬件（如单卡 64G）上跑通更大模型或更长序列
- **手段**：开 offload、调 micro_batch_size、调 gpu_memory_utilization、开 gradient checkpointing 等

### 4. 吞吐优化（正式训练）
- **目标**：最大化训练速度（tokens/s），显存打满但不 OOM
- **手段**：增大 batch_size、调并行策略、balance 推理/训练显存配比

### 5. 分布式扩展
- **目标**：从单卡扩展到多卡/多节点，确保线性或接近线性加速比

## 希望得到的内容

以 `verl.trainer.main_ppo` 的参数为基础，按上述场景整理：

| 场景 | 关键参数 | 调整方向 | 预期效果 |
|---|---|---|---|
| 冒烟测试 | ... | ... | 2 分钟跑通 |
| Overfit 测试 | ... | ... | loss 快速下降 |
| 显存调优 | ... | ... | 显存占用降低 |
| 吞吐优化 | ... | ... | tokens/s 最大 |
| 分布式 | ... | ... | 多卡加速比 |

参数之间的关系和调参优先级：

- `data.train_batch_size` / `ppo_mini_batch_size` / `ppo_micro_batch_size_per_gpu`
- `gpu_memory_utilization` 怎么和 offload / gradient_checkpointing 配合等
- 不同的硬件选型 如何做针对性调整

**Comment by tardis-key** [comment]
关于脚本参数，verl主要负责维护成熟可用的典型脚本，应该不会包含面向不同实验场景的不同配套参数或者方法论。

一方面因为面向实验场景大家需求各不相同，想要达到一定的覆盖率会让脚本数量非常庞大。
另一方面，如何在特性场景下，验证自己想验证的东西，依赖调试人员的经验。难以形成明确的方法论。后续如果有相关的优秀总结，也欢迎提文档的pr。
